docs/content/latest/yugabyte-cloud/cloud-quickstart/qs-explore.md

```diff
@@ -14,211 +14,56 @@ isTocNested: true
-  4 | Arnold Adams       | adams.arnold@gmail.com       | 22 | 49.0560710142838
- 15 | Bertrand Romaguera | romaguera.bertrand@gmail.com | 76 | 28.0989026289413
-  1 | Hudson Borer       | borer-hudson@yahoo.com       |  9 | 81.6742695904106
- 10 | Tressa White       | white.tressa@yahoo.com       | 54 | 122.116378514938
-  4 | Arnold Adams       | adams.arnold@gmail.com       | 23 | 56.5115886738793
-(5 rows)
-```
-
-### Distributed transactions
-
-To track quantities accurately, each product being ordered in some quantity by a user has to decrement the corresponding product inventory quantity. These operations should be performed inside a transaction.
-
-Imagine the user with id `1` wants to order `10` units of the product with id `2`.
-
-Before running the transaction, you can verify the quantity of product `2` in stock by running the following query:
-
-```sql
-yb_demo=# SELECT id, category, price, quantity FROM products WHERE id=2;
-```
-
-```output
-SELECT id, category, price, quantity FROM products WHERE id=2;
- id | category  |      price       | quantity
-----+-----------+------------------+----------
-  2 | Doohickey | 70.0798961307176 |     5000
-(1 row)
-```
-
-To place the order, you can run the following transaction:
-
-```sql
-yb_demo=# BEGIN TRANSACTION;
-
-/* First insert a new order into the orders table. */
-INSERT INTO orders
-  (id, created_at, user_id, product_id, discount, quantity, subtotal, tax, total)
-VALUES (
-  (SELECT max(id)+1 FROM orders)                 /* id */,
-  now()                                          /* created_at */,
-  1                                              /* user_id */,
-  2                                              /* product_id */, 
-  0                                              /* discount */,
-  10                                             /* quantity */,
-  (10 * (SELECT price FROM products WHERE id=2)) /* subtotal */,
-  0                                              /* tax */,
-  (10 * (SELECT price FROM products WHERE id=2)) /* total */
-) RETURNING id;
-
-/* Next decrement the total quantity from the products table. */
-UPDATE products SET quantity = quantity - 10 WHERE id = 2;
-
-COMMIT;
-```
-
-Verify that the order got inserted by running the following command:
-
-```sql
-yb_demo=# SELECT * FROM orders WHERE id = (SELECT max(id) FROM orders);
-```
-
-```output
- id |         created_at         | user_id | product_id | discount | quantity |     subtotal     | tax |      total       
-----+----------------------------+---------+------------+----------+----------+------------------+-----+------------------
- 77 | 2021-09-08 20:03:12.308302 |       1 |          2 |        0 |       10 | 700.798961307176 |   0 | 700.798961307176
-(1 row)
-```
-
-To verify that total quantity of product id `2` in the inventory has been updated, run the following query:
+To create a database, enter the following command:
```

Comment:
Very minor , but I see the yugabyte prompt like this in Cloud Shell : yugabyte=> instead of yugabyte=#